src/components/StructuredList/StructuredList.jsx

```diff
@@ -67,7 +71,7 @@ const StyledStructuredListCell = styled(StructuredListCell)`
  */
 const StructuredList = ({ columns, data, design, isFixedWidth, onRowClick, loadingDataLabel }) => (
   <Fragment>
-    <StructuredListWrapperStyled selection isFixedWidth={isFixedWidth}>
+    <StructuredListWrapperStyled selection isfixedwidth={isFixedWidth.toString()}>
```

Comment:
What was the react error you mention in the PR? The previous solution looks like it should work just fine.

Reply:
<img width="935" alt="image" src="https://user-images.githubusercontent.com/22034430/56929495-2475d080-6ab0-11e9-81ca-7352d7ea31e6.png">

That's why I included `toString()` and the at the StructuredListWrapperStyled the validation is `=== "true"`

Is there a better way?

Reply:
@cgirani as a way to omit custom props from being rendered down on the DOM, you could do something like this:

```
const FilteredHeader = ({myProp, ...otherProps}) => <Header {...otherProps} />;

const StyledHeader = styled(FilteredHeader)`
  background-color: ${ props => props.myProp ? 'green' : 'red' };
`;
```

Reply:
So you could do this:

```
const StructuredListWrapperStyled = styled(
  ({ isFixedWidth, ...otherProps }) => <StructuredListWrapper {...otherProps} />
)`
   {
    && {
      width: ${props => (props.isfixedwidth === 'true' ? 'inherit;' : '')};
    }
    background-color: #ffffff;
    margin-bottom: 0;
    .bx--structured-list-th {
      padding-left: 16px;
    }
    .bx--structured-list-td {
      padding-left: 16px;
      line-height: 8px;
    }
  }
`;
```